packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -480,16 +480,13 @@ export async function parseRequestObject(requestObject: IRequestOptions) {
 	}
 
 	// Axios will follow redirects by default, so we simply tell it otherwise if needed.
+	const { method } = requestObject;
 	if (
-		requestObject.followRedirect === false &&
-		((requestObject.method as string | undefined) || 'get').toLowerCase() === 'get'
-	) {
-		axiosConfig.maxRedirects = 0;
-	}
-	if (
-		requestObject.followAllRedirects === false &&
-		((requestObject.method as string | undefined) || 'get').toLowerCase() !== 'get'
+		(requestObject.followRedirect && (!method || method === 'GET' || method === 'HEAD')) ||
+		requestObject.followAllRedirects
```

Comment:
I think you need to check `requestObject.followRedirect !== false` because it should be treated as `true` by default.

Otherwise you're changing the behavior when `followRedirect` is not passed in.

Reply:
good catch 🙇🏽 . fixed.